plugins/lookup/output.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+name: output
+author: Polona Mihalič (@PolonaM)
+version_added: 1.0.0
+short_description: Read state file outputs.
+description:
+    - This lookup returns all outputs or selected output in state file.
+options:
+  _terms:
+    description:
+    - Names of the outputs to return.
+    - If value is set to C(all), all outputs will be returned.
+    required: True
```

Comment:
I'd much prefer this, and everything, to be optional. I would set this one to `required: false` with no default value, where `None` would then mean "all".

Reply:
Updated as suggested